indra/llcommon/tests/bitpack_test.cpp

```diff
@@ -81,25 +73,25 @@ namespace tut
 
         LLBitPack bitunpack(packbuffer, pack_bufsize*8);
         bitunpack.bitUnpack(&unpackbuffer[0], 8);
-        ensure("bitPack: individual unpack: 0", unpackbuffer[0] == (U8) str[0]);
+        CHECK_MESSAGE(unpackbuffer[0] == (U8, "bitPack: individual unpack: 0") str[0]);
         bitunpack.bitUnpack(&unpackbuffer[0], 8);
-        ensure("bitPack: individual unpack: 1", unpackbuffer[0] == (U8) str[1]);
+        CHECK_MESSAGE(unpackbuffer[0] == (U8, "bitPack: individual unpack: 1") str[1]);
         bitunpack.bitUnpack(&unpackbuffer[0], 8);
-        ensure("bitPack: individual unpack: 2", unpackbuffer[0] == (U8) str[2]);
+        CHECK_MESSAGE(unpackbuffer[0] == (U8, "bitPack: individual unpack: 2") str[2]);
         bitunpack.bitUnpack(&unpackbuffer[0], 8);
-        ensure("bitPack: individual unpack: 3", unpackbuffer[0] == (U8) str[3]);
+        CHECK_MESSAGE(unpackbuffer[0] == (U8, "bitPack: individual unpack: 3") str[3]);
         bitunpack.bitUnpack(&unpackbuffer[0], 8);
-        ensure("bitPack: individual unpack: 4", unpackbuffer[0] == (U8) str[4]);
+        CHECK_MESSAGE(unpackbuffer[0] == (U8, "bitPack: individual unpack: 4") str[4]);
         bitunpack.bitUnpack(&unpackbuffer[0], 8);
-        ensure("bitPack: individual unpack: 5", unpackbuffer[0] == (U8) str[5]);
+        CHECK_MESSAGE(unpackbuffer[0] == (U8, "bitPack: individual unpack: 5") str[5]);
         bitunpack.bitUnpack(unpackbuffer, 8*4); // Life
         ensure_memory_matches("bitPack: 4 bytes unpack:", unpackbuffer, 4, str+6, 4);
-    }
+    
+}
+
+TEST_CASE("test_3")
```

Comment:
These test cases could receive more descriptive names (Ex. in this case "U32 packing").